src/makeInfinityAdder.js

```diff
@@ -6,7 +6,29 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let adderInput = [0];
+
+  const adder = (n) => {
+    if (n === undefined) {
+      const res = adderInput.reduce((total, el) => {
+        return total + el;
+      }, 0);
+
+      adderInput = [0];
+
+      return res;
+    };
+
+    adderInput.push(n);
```

Comment:
if you'll just add the number to the adderInput you don't need to use reduce. reduce is overkill for this task)